x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -22,23 +22,23 @@ import {
 import { esKuery } from '../../../../../src/plugins/data/server';
 import { ActionsClient, ActionsAuthorization } from '../../../actions/server';
 import {
-  Alert,
-  PartialAlert,
+  Alert as Rule,
```

Comment:
Because of the change from `AlertInstance` to `Alert`, I had to update these to `Rule`. We will eventually rename them for real.